### ISSUE TITLE
Dinger not recognized as PWA by Lighthouse

### ISSUE THREAD

**Issue by ethanwlee** [TBD54566975/dinger#4]
## Description
Running the [Google Lighthouse](https://github.com/GoogleChrome/lighthouse) audit on the [dinger website](https://dinger.web5.website/) leads to an invalid PWA score (other performance scores on Lighthouse are calculated on a scale of 0-100, but the [PWA score is evaluated differently](https://github.com/GoogleChrome/lighthouse/blob/main/docs/scoring.md#how-is-the-pwa-progressive-web-app-score-calculated)). This seems to contradict the fact that the dinger website is seemingly installable as a PWA.

## Screenshot of Issue
![Screenshot 2023-05-31 at 21 37 08](https://github.com/TBD54566975/dinger/assets/125412902/e7c8f26a-3730-4469-9e09-7e56793984a8)

## Reproduce the Issue 
Run the Lighthouse audit by right-clicking on the dinger webpage and clicking `Inspect`. Then, select the `Lighthouse` tab in the DevTools panel that pops up (may need to hit `>>`) and hit `Analyze page load`.

**Comment by ethanwlee** [comment]
cc @michaelneale

**Comment by michaelneale** [comment]
interesting @ethan-tbd - yeah I found another site which did this but didn't realise was in chrome itself, neat. Yes I am not surprised, to really be a PWA should use a service worker, but it doesn't right now (doesn't really need to). I have opened a PR that adds it: https://deploy-preview-5--chipper-meerkat-51825b.netlify.app/ - doesn't really do anything with it yet, but worth trying to see if it works for you. I'm not sure if I have set it up right though.
